pedalboard/RubberbandPlugin.h

```diff
@@ -0,0 +1,87 @@
+#include "../vendors/rubberband/rubberband/RubberBandStretcher.h"
+#include "Plugin.h"
+
+using namespace RubberBand;
+
+namespace Pedalboard {
+class RubberbandPlugin : public Plugin
+/*
+Base class for rubberband plugins.
+*/
+{
+public:
+  virtual ~RubberbandPlugin(){};
+
+  void process(
+      const juce::dsp::ProcessContextReplacing<float> &context) override final {
+    if (rbPtr) {
+      auto inBlock = context.getInputBlock();
+      auto outBlock = context.getOutputBlock();
+
+      auto len = inBlock.getNumSamples();
+      auto numChannels = inBlock.getNumChannels();
+
+      jassert(len == outBlock.getNumSamples());
+      jassert(numChannels == outBlock.getNumChannels());
+
+      const float **inChannels =
+          (const float **)alloca(numChannels * sizeof(float *));
```

Comment:
Any particular reason as to why you're using alloca and not a `std::vector<float*>`  or a float[][]? alloca allocates on the stack and that can be dangerous in some cases.

But also do you know if the samples or channels will be constant throughout the use of this plugin? If so you should probably preallocate this in the constructor.